component/sections/pricing/index.js

```diff
@@ -14,9 +20,19 @@ function Pricing({ data }) {
     monthlyBilling: data?.variants?.[variant]?.monthlyBilling,
     banner: data?.variants?.[variant]?.banner,
     form: data?.variants?.[variant]?.form,
+    stripePKey: data?.variants?.[variant]?.stripeAccount?.stripePKey,
```

Comment:
Not sure but do we still need this `stripePKey` ?

Reply:
Yes, for checkout button.